Bottom: 048d046cf14a4ace0f6bcacac30b04a99f4015a6
Top:    831a2effddf22de1aad3e8c28b2567874552b05b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-31 02:17:12 +0100

Refresh of add-strip-hierarchy-options-to

---

diff --git a/src/action.c b/src/action.c
index c7316e5..e891726 100644
--- a/src/action.c
+++ b/src/action.c
@@ -4833,7 +4833,6 @@ ActionToggleStripHierarchy (int argc, char **argv, int x, int y)
   char *function = ARG (0);
   if (function && PCB->ElementOn)
     {
-      HideCrosshair (true);
       switch (GetFunctionID (function))
 	{
 	case F_Object:
@@ -4883,7 +4882,6 @@ ActionToggleStripHierarchy (int argc, char **argv, int x, int y)
 	      }
 	  }
 	}
-      RestoreCrosshair (true);
     }
   return 0;
 }
